src/testUtils/mocks/dataSources.ts

```diff
@@ -1578,3 +1578,6 @@ export const createMockCanTransferResult = (
 export const setRuntimeVersion = (args: unknown): void => {
   mockInstanceContainer.apiInstance.runtimeVersion = args as RuntimeVersion;
 };
+
+export const createMockText = (value: string): Text =>
+  (createMockStringCodec(value) as unknown) as Text;
```

Comment:
I think the `as unknown` isn't necessary

Reply:
If I remove the cast: `Conversion of type 'Codec' to type 'Text' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`